pretix_saferpay/payment.py

```diff
@@ -789,22 +789,14 @@ class SaferpayEPS(SaferpayMethod):
     payment_methods = ["EPS"]
 
 
-class SaferpayGiropay(SaferpayMethod):
+class SaferpayGiropay(SaferpayMethod, RetiredMethodMixin):
```

Comment:
Mixins before base-classes! Same for the others
```suggestion
class SaferpayGiropay(RetiredMethodMixin, SaferpayMethod):
```